bundles/automation/org.eclipse.smarthome.automation.module.script.extensions.rulesupport/ESH-INF/automation/moduletypes/PrivateScriptedTypes.json

```diff
@@ -0,0 +1,41 @@
+{  
```

Comment:
Why do you need these types and cannot simply use https://github.com/eclipse/smarthome/blob/master/bundles/automation/org.eclipse.smarthome.automation.module.script/ESH-INF/automation/moduletypes/ScriptTypes.json?